### PR TITLE
update: added list component

### PR DESCRIPTION
**Pull request by simran61**
Added list components

This has the following lists:

1.  ordered list
2. unordered list
3. spaced list
4. stacked list

### CODE REVIEW

File: components/Lists/lists.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
+.show_list{
+    width: 700px;
```
Comment (znjs) [inline components/Lists/lists.css:2]: Use relative units if possible

Comment (Nishant-96) [inline components/Lists/lists.css:2]: Suggestion: Try to use relative units.

File: components/Lists/lists.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+.list_style{
+    color: black;
+}
+
+.customized_list{
+    width: 80%;
+    border-left: 1px solid grey;
+    border-right: 1px solid grey;
```
Comment (znjs) [inline components/Lists/lists.css:17]: can change into single border style

File: components/Lists/lists.css [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    color: black;
+}
+
+.list_style{
+    color: black;
+}
+
+.customized_list{
+    width: 80%;
+    border-left: 1px solid grey;
```
Comment (Nishant-96) [inline components/Lists/lists.css:16]: you can use shorthand for bottom.

**Review by znjs — APPROVED** [review]

**Review by Nishant-96 — APPROVED** [review]

**Pull request merged**